project/AkkaBuild.scala

```diff
@@ -64,7 +64,9 @@ object AkkaBuild {
     UnidocRoot.akkaSettings,
     Protobuf.settings,
     parallelExecution in GlobalScope := System.getProperty("akka.parallelExecution", parallelExecutionByDefault.toString).toBoolean,
-      version in ThisBuild := akkaVersion
+    version in ThisBuild := akkaVersion,
+    // used for linking to API docs (overwrites `project-info.version`)
+    ThisBuild / projectInfoVersion := { if (isSnapshot.value) "snapshot" else version.value }
```

Comment:
is there a problem for SEO to have links to the exact version, or should we prefer minor version number (2.6)?

Reply:
No, as the Akka docs and the APIs are deployed together for each patch version, we are alright, I think.